Composer/packages/client/src/components/Split/ThinSplitter.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { RenderSplitterProps } from '@geoffcox/react-splitter';
+
+type Props = RenderSplitterProps & {
+  color: string;
+  hoverColor: string;
+};
+
+const HitArea = styled.div(({ horizontal, hoverColor }: { horizontal: boolean; hoverColor: string }) => ({
+  boxSizing: 'border-box',
+  outline: 'none',
+  overflow: 'hidden',
+  height: '100%',
+  width: '100%',
+  cursor: horizontal ? 'row-resize' : 'col-resize',
+  background: 'transparent',
+  '%:hover .thin-split-visual': {
```

Comment:
is this valid syntax? % ? shouldn't it be &?

Reply:
Yeah - that is weird.  Will fix.